package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "npx standard index.js",
-    "package": "npx ncc build index.js -m -o dist"
+    "package": "NODE_OPTIONS=--openssl-legacy-provider npx ncc build index.js -m -o dist"
```

Comment:
Updating to node18 resulted in [this error](https://github.com/byu-oit/github-action-create-ecr-repo-if-missing/actions/runs/3990547854/jobs/6844321537#step:5:13):
```
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:[13](https://github.com/byu-oit/github-action-create-ecr-repo-if-missing/actions/runs/3990547854/jobs/6844321537#step:5:14)3:10)
    at hashOf (/home/runner/work/github-action-create-ecr-repo-if-missing/github-action-create-ecr-repo-if-missing/node_modules/@zeit/ncc/dist/ncc/index.js.cache.js:3:58216)
    at module.exports (/home/runner/work/github-action-create-ecr-repo-if-missing/github-action-create-ecr-repo-if-missing/node_modules/@zeit/ncc/dist/ncc/index.js.cache.js:3:60642)
    at runCmd (/home/runner/work/github-action-create-ecr-repo-if-missing/github-action-create-ecr-repo-if-missing/node_modules/@zeit/ncc/dist/ncc/cli.js.cache.js:1:47355)
    at 819 (/home/runner/work/github-action-create-ecr-repo-if-missing/github-action-create-ecr-repo-if-missing/node_modules/@zeit/ncc/dist/ncc/cli.js.cache.js:1:44227)
    at __webpack_require__ (/home/runner/work/github-action-create-ecr-repo-if-missing/github-action-create-ecr-repo-if-missing/node_modules/@zeit/ncc/dist/ncc/cli.js.cache.js:1:[16](https://github.com/byu-oit/github-action-create-ecr-repo-if-missing/actions/runs/3990547854/jobs/6844321537#step:5:17)9)
    at startup (/home/runner/work/github-action-create-ecr-repo-if-missing/github-action-create-ecr-repo-if-missing/node_modules/@zeit/ncc/dist/ncc/cli.js.cache.js:1:339)
    at module.exports.8 (/home/runner/work/github-action-create-ecr-repo-if-missing/github-action-create-ecr-repo-if-missing/node_modules/@zeit/ncc/dist/ncc/cli.js.cache.js:1:371)
    at /home/runner/work/github-action-create-ecr-repo-if-missing/github-action-create-ecr-repo-if-missing/node_modules/@zeit/ncc/dist/ncc/cli.js.cache.js:1:381 {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
```

I didn't get that error when running locally on the same version of Node, but a newer version of NPM. I don't see any way to specify the version of NPM in the `setup-node` action.

This `NODE_OPTIONS` was the only way I could find to get past this error. I'm open to suggestions for a better solution.

Reply:
https://github.com/byu-oit/appeng-questions/discussions/17
and
https://github.com/byu-oit/appeng-questions/discussions/42
both seem related. I guess this is the best workaround for now.